.github/workflows/ci.yml

```diff
@@ -77,6 +77,9 @@ jobs:
           repository: ${{ inputs.repository || github.repository }}
           ref: ${{ inputs.ref }}
           submodules: recursive
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
```

Comment:
Pulled out into a separate PR - https://github.com/desktop/desktop/pull/17641

But, we need this for our runners to work to release.